portfolio/src/main/webapp/book-detail/book-detail.template.html

```diff
@@ -25,6 +25,19 @@ <h2>{{bookDetailCtrl.book.title}}</h2>
                 <a ng-href="#!/book-detail/{{eachBook.isbn}}">{{eachBook.title}}</a> 
             </li>
         </p>
+
+        <p class="booklst">Reviews: </p> 
```

Comment:
Is it possible to hide this form when the user has already left a review for this book? You can do this in separate PR.